apps/ledger-live-mobile/src/analytics/segment.ts

```diff
@@ -91,45 +97,43 @@ const extraProperties = store => {
         }
       : {}),
     ...deviceInfo,
+    notificationsBlacklisted,
   };
 };
 
 const context = {
   ip: "0.0.0.0",
 };
 let storeInstance; // is the redux store. it's also used as a flag to know if analytics is on or off.
+let segmentClient: SegmentClient | undefined;
 
-const token = __DEV__ ? null : ANALYTICS_TOKEN;
-export const start = async (store: any) => {
-  if (token) {
-    await analytics.setup(token, {
-      android: {
-        collectDeviceId: false,
-      },
-      ios: {
-        trackAdvertising: false,
-        trackDeepLinks: false,
-      },
-    });
-  }
-
+const token = ANALYTICS_TOKEN;
+export const start = async (store: any): Promise<SegmentClient | undefined> => {
   const { user, created } = await getOrCreateUser();
   storeInstance = store;
 
-  if (ANALYTICS_LOGS) console.log("analytics:identify", user.id);
+  if (created && ANALYTICS_LOGS) {
+    console.log("analytics:identify", user.id);
```

Comment:
Isn't that console.log causing a TS issue ?

Reply:
No, we've got a `/* eslint-disable no-console */` at the start of the file